mods/d2k/rules/infantry.yaml

```diff
@@ -108,3 +107,124 @@ MEDIC:
 	AttractsWorms:
 		Intensity: 180
 
+fremen:
+	Inherits: ^Infantry
+	Valued:
+		Cost: 800
+	Tooltip:
+		Name: Fremen
+		Description: Elite sniper infantry unit\n  Strong vs Infantry\n  Weak vs Vehicles\n  Special Ability: Invisibility
+	Buildable:
+		Queue: Infantry
+		BuildPaletteOrder: 85
+		Prerequisites: ~barr.atreides, palace, ~techlevel.high
```

Comment:
There are two choices if we want that:

1.  Have the build icons hidden until you build the appropriate palace (inconsistent with everything else).
2.  Allow players to always be able to see/build their own faction-specific units, even if they don't own their own faction's palace.